src/plugins/bodhi.c

```diff
@@ -335,6 +335,7 @@ static GHashTable *bodhi_parse_json(json_object *json, const char *release)
             continue;
 
         int karma, unstable_karma;
+        karma = unstable_karma = 0;
```

Comment:
Please, *please*, **please** something like this, instead:

```c
int karma = 0;
int unstable_karma = 0;
```

Reply:
sorry :roll_eyes:

Reply:
karma++